src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -77,6 +77,7 @@ public static bool IsDrawingSupported
         }
 
         public static bool IsInContainer => GetIsInContainer();
+        public static bool SupportsComInterop => IsWindows && IsNetCore; // matches definitions in clr.featuredefines.props
```

Comment:
`IsNetCore` is true for everything except .NET Framework.

I think this should be `IsWindows && !IsMonoRuntime`.